package.json

```diff
@@ -34,27 +30,18 @@
     "victory-util": "^2.0.2"
   },
   "devDependencies": {
-    "babel": "^5.5.8",
-    "babel-core": "^5.5.8",
-    "babel-loader": "^5.3.2",
-    "builder-react-component-dev": "~0.1.1",
+    "builder-victory-component-dev": "~0.0.4",
     "chai": "^3.2.0",
-    "clean-webpack-plugin": "^0.1.4",
-    "docgen-loader": "1.3.2",
     "ecology": "~1.0.4",
     "formidable-landers": "latest",
     "mocha": "^2.2.5",
     "raw-loader": "^0.5.1",
-    "react": "0.14.x",
+    "react": "0.14.x",  
     "react-addons-test-utils": "0.14.x",
     "react-docgen": "^2.4.0",
     "react-dom": "0.14.x",
     "react-hot-loader": "^1.2.8",
```

Comment:
`react-hot-loader` can be removed, since it's in `builder-victory-component`'s dev/package.json.